<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>리뷰 보기</string>
  </property>
  <widget class="QLabel" name="reviewRestLable">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="reviewList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>561</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
